storage/innobase/xtrabackup/src/backup_mysql.cc

```diff
@@ -189,13 +189,14 @@ xb_mysql_connect()
 	xb_mysql_query(connection, "SET NAMES utf8",
 		       false, true);
 
-	if (xb_mysql_numrows(connection, "SELECT @@wsrep_sync_wait", false) >
-				0) {
-		xb_mysql_query(connection, "SET SESSION wsrep_sync_wait=0", false,
-				true);
-	}
-
-	return(connection);
+        if (xb_mysql_numrows(connection,
+                             "SHOW GLOBAL VARIABLES LIKE '%wsrep_sync_wait%'",
```

Comment:
Please remove %. We know the exact variable name and we can do strict match

Reply:
fixed